kratos/containers/variables_list.h

```diff
@@ -263,6 +267,40 @@ namespace Kratos
 			mDataSize += static_cast<SizeType>(((block_size - 1) + ThisVariable.Size()) / block_size);
 		}
 
+		int AddDof(VariableData const* pThisDofVariable, VariableData const* pThisDofReaction){
+			
+			for(std::size_t dof_index = 0 ; dof_index < mDofVriables.size() ; dof_index++){
+				if(*mDofVriables[dof_index] == *pThisDofVariable){
+					mDofReactions[dof_index] = pThisDofReaction;
+					return static_cast<int>(dof_index);
+				}
+			}
+
+#ifdef KRATOS_DEBUG
+        if(OpenMPUtils::IsInParallel() != 0)
+            KRATOS_ERROR << "attempting to call AddDof for: " << pThisDofVariable << ". Unfortunately the Dof was not added before and the operations is not threadsafe (this function is being called within a parallel region)" << std::endl;
```

Comment:
Don't you prefer to use KRATOS_ERROR_IF directly?, it is clearer

Reply:
It was to avoid the include in release mode as it was before. You are right that the other way is preferred.

Reply:
I think he meant `KRATOS_ERROR_IF(OpenMPUtils::IsInParallel() != 0)`

Reply:
I know, I just didn't want to change how the omp_utils.h is included.